bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -178,24 +178,42 @@ export class PlotCanvasView extends BokehView
     # Update any DataRange1ds here
     frame = @model.frame
     bounds = {}
+    log_bounds = {}
+
+    calculate_log_bounds = false
+    for r in _.values(frame.x_ranges).concat(_.values(frame.y_ranges))
+      if r instanceof DataRange1d
+        if r._mapper_type == "log"
```

Comment:
You use `@internal` when you don't want to propagate changes from bokehjs to bokeh. This way, an internal property in bokehjs doesn't have to have its mirror in bokeh. In this case you want to use `@internal`.